jsonpatch.py

```diff
@@ -800,7 +802,8 @@ def _item_removed(self, path, key, item):
         new_index = self.insert(new_op)
         if index is not None:
             op = index[2]
-            if type(op.key) == int:
+            added_item = op.pointer.to_last(self.dst_doc)[0]
+            if type(added_item) == list:
```

Comment:
Could you please just add a comment here to clarify for posterity?

Reply:
I've done so now. Please let me know if there's anything else I should add in the comment. It was a bit difficult to express what the _on_undo_add call accomplishes and I think it'd be valuable to mention that.

Reply:
@stefankoegl kind reminder on this